pkg/tools/assessmentopts.go

```diff
@@ -130,12 +131,18 @@ func (o *AssessmentOpts) GetCustomPoliciesDir(policyTypeName string, morePolicyT
 	dir := o.CustomPoliciesDir
 
 	if dir == "" {
-		url := fmt.Sprintf("/api/v1/org/{org}/policies/%s/policies.tgz", o.Tool.Name())
+		url := fmt.Sprintf("/api/v1/org/{org}/policies/%s/policies.zip", o.Tool.Name())
 		d, err := o.InstallAPIServerArtifact(fmt.Sprintf("%s-%s-policies", o.Tool.Name(),
 			api.Organization), url)
 		if err != nil {
 			return "", err
 		}
+		if d.StatusCode == http.StatusNoContent {
```

Comment:
I think it would be better to return an error from Download() and not return the resty response.  You can define an ErrNoContent has check for that.

Reply:
Done 👍